### PR TITLE
aWATTar: Add support for aWATTar Germany

### PR DESCRIPTION
**Pull request by mzanetti**
nymea-plugins pull request checklist:

- [x] Make sure the pull request's title is of format "Plugin name: Add support for xyz" or "New plugin: Plugin name"

- [x] Did you test the changes on hardware, if not (e.g. absence of required hardware), please mention a person to confirm it has been tested.

- [x] Did you update the plugin's README.md accordingly?

- [x] Did you update translations (`cd builddir && make lupdate`)?

- [x] If you added a new plugin, should it be added to nymea-plugins-all or nymea-plugins-maker?

### CODE REVIEW

File: awattar/integrationpluginawattar.cpp [hunk 1]
```diff
@@ -116,7 +139,7 @@ void IntegrationPluginAwattar::requestPriceData(Thing* thing, ThingSetupInfo *se
```
Comment (Boernsman) [inline awattar/integrationpluginawattar.cpp]: m_pluginTimer isn't set to nullptr after unregistering the timer, so after readding the device the timer won't work

File: awattar/integrationpluginawattar.json [hunk 2]
```diff
@@ -89,6 +90,79 @@
...
+                            "displayName": "Online",
+                            "displayNameEvent": "Online changed",
+                            "type": "bool",
+                            "defaultValue": false,
+                            "cached": false
+                        },
+                        {
+                            "id": "048566fe-d49d-4cc7-b93b-520f727f600a",
+                            "name": "currentMarketPrice",
+                            "displayName": "current market price",
```
[the remark this answers is not in the document]
Comment (mzanetti) [reply awattar/integrationpluginawattar.json]: yeah, copied that from the AT one, I've updated both now, DE and AT

File: awattar/integrationpluginawattar.h [hunk 3]
```diff
@@ -57,13 +57,22 @@ class IntegrationPluginAwattar : public IntegrationPlugin
     void setupThing(ThingSetupInfo *info) override;
```
[the remark this answers is not in the document]
Comment (mzanetti) [reply awattar/integrationpluginawattar.h]: ack

File: awattar/integrationpluginawattar.cpp (hunk 1 above)
Comment (mzanetti) [reply awattar/integrationpluginawattar.cpp]: good catch! Not something I introduced in this branch, but I still fixed it now

**Comment by mzanetti** [comment]
landedd

**Pull request closed without merging**